SPECS/igc-module.spec

```diff
@@ -0,0 +1,54 @@
+%define uname  %{kernel_version}
```

Comment:
We usually don't do this and just use %{kernel_version} which is here for this. The only examples of such defines are in your dell-rbu-module and coretemp-module-alt RPMs, but I didn't notice at the time.